testutils/clustermanager/gke.go

```diff
@@ -36,6 +38,8 @@ type GKECluster struct {
 	NeedCleanup bool
 	// TODO: evaluate returning "google.golang.org/api/container/v1.Cluster" when implementing the creation logic
 	Cluster *string
+
+	Exec func(string, ...string) ([]byte, error)
```

Comment:
I know what you're doing, but this surely deserves a comment. :)

Reply:
done

Reply:
Shouldn't you explicitly say it's intended mostly for testing purposes? Right now I can't imagine any other reason...